code/modules/core_implant/cruciform/machinery/eotp.dm

```diff
@@ -53,7 +53,7 @@ var/global/obj/machinery/power/eotp/eotp
 /obj/machinery/power/eotp/New()
 	..()
 	eotp = src
-	START_PROCESSING(SSobj, src)
+	START_PROCESSING(SSmachines, src)
```

Comment:
I think this is unnecessary since all machines start SSmachines when initialized.

Reply:
Change already done. You can see in commits. (last one)

Reply:
Are you sure?

Reply:
![image](https://user-images.githubusercontent.com/49106866/102415780-2535e000-3fcf-11eb-9d46-eb920a82b532.png)

Reply:
Okay, then i will test this.

Reply:
Yep, as you say.

Reply:
Changes done.